src/test/ui/issues/issue-72574-2.rs

```diff
@@ -0,0 +1,11 @@
+struct Binder(i32, i32, i32);
+
+fn main() {
+    let x = Binder(1, 2, 3);
+    match x {
+        Binder(_a, _x @ ..) => {}
+        _ => {}
+    }
+}
+//~^^^^ ERROR `_x @` is not allowed in a tuple struct
+//~^^^^^ ERROR this pattern has 1 field, but the corresponding tuple struct has 3 fields [E0023]
```

Comment:
Same here.

Reply:
Will change this once I have a look at the errors following your recommended source changes.

Reply:
This goes away with the changes that you recommended in https://github.com/rust-lang/rust/pull/72677#discussion_r431964600 too. 👍

Reply:
https://github.com/rust-lang/rust/pull/72677/commits/e73ac086c54084dced1916d062267be4058b257c